### PR TITLE
attempting to demystify the mysterious

### PR DESCRIPTION
**Pull request by theophani**
My attempt to clarify the mysterious CSS positioning. Any thoughts before I merge?
/cc @marijnh @molily

### CODE REVIEW

File: page2.html [hunk 1]
```diff
@@ -238,22 +238,49 @@ <h3 class=ex>Explanation</h3>
 <code>class-name</code> as their class attribute.</p>
 
 <p>By using class names, we can control two <code>&lt;div></code> tags
-in completely different ways, and we can use this control to now we
-can see that CSS can also control position.</p>
+in completely different ways, and we can also use this control to
+control the position of elements.</p>
```
Comment (knuton) [inline page2.html]: > we can use this control to control

Is this poetic construction intentional?

Comment (theophani) [inline page2.html]: Sort of? but it is more awkward than anything else. Maybe just "use this control to position elements"

File: page2.html [hunk 2]
```diff
@@ -238,22 +238,49 @@ <h3 class=ex>Explanation</h3>
...
-strip of all our photos. By changing the position of the film roll, we
-can make our photos slide back and forth inside the picture frame.</p>
+<aside>
+<p>Nerdy details: CSS positioning is an advanced topic. Hopefully,
+it is enough to say:</p>
+<ul>
+  <li><code>margin: 50px auto;</code> makes the frame horizontally centered</li>
+  <li><code>overflow: hidden;</code> hide contents that don’t fit inside the frame</li>
+  <li><code>position: relative;</code> allows us to have a frame of reference to position any items inside</li>
+  <li><code>position: absolute;</code> allows us to set the position</li>
```
Comment (theophani) [inline page2.html:281]: By the way, I intentionally left out mentioning <code>float: left</code> on images. It's needed to force the images to touch each other to overcome their natural inline flow behaviour. I could add a line about this.

Comment (marijnh) [inline page2.html:281]: But wouldn't it also be viable to use the inline-block nature of images to simply cram them next to each other and lose a lot of the css?

_(Disclaimer: I didn't read the current version of the text closely yet, so I'm probably missing several reasons why you did things this way.)_

Comment (theophani) [inline page2.html:281]: With inline-block they have gaps between unless we also force the img tags to touch, and I'd have to then explain THAT instead. (The only CSS on the images is the float:left, to force them to touch.)

Comment (marijnh) [inline page2.html:281]: Fair enough!

Comment (theophani) [inline page2.html:281]: CSS is magic. Black magic.

**Pull request merged**